diff --git a/CMakeLists.txt b/CMakeLists.txt
index 37b19d6b..623f23f0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -422,5 +422,5 @@ add_executable(cstool ${CSTOOL_SRC})
 target_link_libraries(cstool ${default-target})
 
 install(TARGETS cstool DESTINATION bin)
-install(FILES ${CMAKE_BINARY_DIR}/capstone.pc DESTINATION lib/pkgconfig)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/capstone.pc DESTINATION lib/pkgconfig)
 endif ()
